modules/op-configuring-ssh-authentication-for-git.adoc

```diff
@@ -0,0 +1,97 @@
+// This module is included in the following assembly:
+//
+// *openshift-docs/cicd/pipelines/authenticating-pipelines-using-git-secret.adoc
+
+[id="op-configuring-ssh-authentication-for-git_{context}"]
+= Configuring SSH authentication for Git
+
+For SSH authentication, {pipelines-shortname} creates a `~/.ssh/config` file containing the SSH private key specified in the secret, before executing any steps in a task run or a pipeline run. When the steps execute, {pipelines-shortname} uses the key to retrieve the pipeline resources specified in a run.
```

Comment:
Please provide an abstract and summarize the procedure in terms of the user, not the system.

Reply:
The purpose of this paragraph in this module and in the "basic auth" module is to tell the user very briefly what's going on in the background. It's primarily educational in nature, and throws light on the reason behind some steps in the procedure. There is basically nothing that the user can do about it.